src/main/java/org/jenkinsci/plugins/p4/scm/SwarmScmSource.java

```diff
@@ -125,7 +143,7 @@ public P4Browser getBrowser() {
 			// Get first Swarm path; it MUST include the Jenkinsfile
 			P4Path p4Path = branch.getPath();
 
-			P4SCMHead head = new P4SCMHead(branch.getId(), p4Path);
+			P4SCMHead head = new P4SCMHead(branch.getName(), p4Path);
```

Comment:
(reminder for me to check all uses of getId() for case sensitivity)
#PR124 addresses another instance of getId() / getName()